### PR TITLE
Implement dumb invites page component

### PR DESCRIPTION
**Pull request by chromakode**
:eyeglasses: @keybase/react-hackers

### CODE REVIEW

**Comment by keybase-ci-visdiff** [comment]
:mag_right: The commits 41f9d38f55dd...922211cd6f8d introduced some visual changes on linux:
 * Added: **Invites-Normal_Empty_Message**  
   Invites-Normal_Empty_Message rendered
 * Added: **Invites-Normal_Empty**  
   Invites-Normal_Empty rendered
 * Added: **Invites-Normal_Email_Error**  
   Invites-Normal_Email_Error rendered
 * Added: **Invites-Normal**  
   Invites-Normal rendered
 * Changed: **Landing-Normal_Not_Verified_email** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1474671311597-41f9d38f55dd-922211cd6f8d-linux/Landing-Normal_Not_Verified_email-41f9d38f55dd.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1474671311597-41f9d38f55dd-922211cd6f8d-linux/Landing-Normal_Not_Verified_email-922211cd6f8d.png)  
   Landing-Normal_Not_Verified_email comparison
 * Changed: **Landing-Normal** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1474671311597-41f9d38f55dd-922211cd6f8d-linux/Landing-Normal-41f9d38f55dd.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1474671311597-41f9d38f55dd-922211cd6f8d-linux/Landing-Normal-922211cd6f8d.png)  
   Landing-Normal comparison
 * Changed: **Landing-Gold_Plan_Broken_Payment** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1474671311597-41f9d38f55dd-922211cd6f8d-linux/Landing-Gold_Plan_Broken_Payment-41f9d38f55dd.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1474671311597-41f9d38f55dd-922211cd6f8d-linux/Landing-Gold_Plan_Broken_Payment-922211cd6f8d.png)  
   Landing-Gold_Plan_Broken_Payment comparison
 * Changed: **Landing-Gold_Plan** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1474671311597-41f9d38f55dd-922211cd6f8d-linux/Landing-Gold_Plan-41f9d38f55dd.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1474671311597-41f9d38f55dd-922211cd6f8d-linux/Landing-Gold_Plan-922211cd6f8d.png) [(diff)](https://keybase-app-visdiff.s3.amazonaws.com/1474671311597-41f9d38f55dd-922211cd6f8d-linux/Landing-Gold_Plan.png)

**Comment by keybase-ci-visdiff** [comment]
:mag_right: The commits 41f9d38f55dd...ba1cc7dca0bf introduced some visual changes on linux:
 * Added: **Invites-Normal_Empty_Message**  
   Invites-Normal_Empty_Message rendered
 * Added: **Invites-Normal_Empty**  
   Invites-Normal_Empty rendered
 * Added: **Invites-Normal_Email_Error**  
   Invites-Normal_Email_Error rendered
 * Added: **Invites-Normal**  
   Invites-Normal rendered
 * Changed: **Landing-Normal_Not_Verified_email** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1474672812900-41f9d38f55dd-ba1cc7dca0bf-linux/Landing-Normal_Not_Verified_email-41f9d38f55dd.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1474672812900-41f9d38f55dd-ba1cc7dca0bf-linux/Landing-Normal_Not_Verified_email-ba1cc7dca0bf.png)  
   Landing-Normal_Not_Verified_email comparison
 * Changed: **Landing-Normal** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1474672812900-41f9d38f55dd-ba1cc7dca0bf-linux/Landing-Normal-41f9d38f55dd.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1474672812900-41f9d38f55dd-ba1cc7dca0bf-linux/Landing-Normal-ba1cc7dca0bf.png)  
   Landing-Normal comparison
 * Changed: **Landing-Gold_Plan_Broken_Payment** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1474672812900-41f9d38f55dd-ba1cc7dca0bf-linux/Landing-Gold_Plan_Broken_Payment-41f9d38f55dd.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1474672812900-41f9d38f55dd-ba1cc7dca0bf-linux/Landing-Gold_Plan_Broken_Payment-ba1cc7dca0bf.png)  
   Landing-Gold_Plan_Broken_Payment comparison
 * Changed: **Landing-Gold_Plan** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1474672812900-41f9d38f55dd-ba1cc7dca0bf-linux/Landing-Gold_Plan-41f9d38f55dd.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1474672812900-41f9d38f55dd-ba1cc7dca0bf-linux/Landing-Gold_Plan-ba1cc7dca0bf.png) [(diff)](https://keybase-app-visdiff.s3.amazonaws.com/1474672812900-41f9d38f55dd-ba1cc7dca0bf-linux/Landing-Gold_Plan.png)

**Comment by chromakode** [comment]
@keybase/react-hackers bump

**Comment by keybase-ci-visdiff** [comment]
:mag_right: The commits e4fea7c30ce7...9e869fc542da introduced some visual changes on linux:
 * Added: **Invites-Normal_Empty_Message**  
   Invites-Normal_Empty_Message rendered
 * Added: **Invites-Normal_Empty**  
   Invites-Normal_Empty rendered
 * Added: **Invites-Normal_Email_Error**  
   Invites-Normal_Email_Error rendered
 * Added: **Invites-Normal**  
   Invites-Normal rendered
 * Changed: **Landing-Normal_Not_Verified_email** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1475015932637-e4fea7c30ce7-9e869fc542da-linux/Landing-Normal_Not_Verified_email-e4fea7c30ce7.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1475015932637-e4fea7c30ce7-9e869fc542da-linux/Landing-Normal_Not_Verified_email-9e869fc542da.png)  
   Landing-Normal_Not_Verified_email comparison
 * Changed: **Landing-Normal** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1475015932637-e4fea7c30ce7-9e869fc542da-linux/Landing-Normal-e4fea7c30ce7.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1475015932637-e4fea7c30ce7-9e869fc542da-linux/Landing-Normal-9e869fc542da.png)  
   Landing-Normal comparison
 * Changed: **Landing-Gold_Plan_Broken_Payment** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1475015932637-e4fea7c30ce7-9e869fc542da-linux/Landing-Gold_Plan_Broken_Payment-e4fea7c30ce7.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1475015932637-e4fea7c30ce7-9e869fc542da-linux/Landing-Gold_Plan_Broken_Payment-9e869fc542da.png)  
   Landing-Gold_Plan_Broken_Payment comparison
 * Changed: **Landing-Gold_Plan** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1475015932637-e4fea7c30ce7-9e869fc542da-linux/Landing-Gold_Plan-e4fea7c30ce7.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1475015932637-e4fea7c30ce7-9e869fc542da-linux/Landing-Gold_Plan-9e869fc542da.png) [(diff)](https://keybase-app-visdiff.s3.amazonaws.com/1475015932637-e4fea7c30ce7-9e869fc542da-linux/Landing-Gold_Plan.png)

File: shared/settings/invites/index.desktop.js [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+      </Box>
+    </Box>
+  )
+}
+
+function PendingInviteItem ({invite, onReclaim}: {invite: PendingInvite, onReclaim: (id: string) => void}) {
+  return (
+    <Box style={styleInviteItem}>
+      {invite.type === 'pending-email' ? <PendingEmailContent invite={invite} /> : <PendingURLContent invite={invite} />}
+      <Box style={{flex: 1}} />
```
Comment (chrisnojima) [inline shared/settings/invites/index.desktop.js]: not to change anything, but we could also use space-between and not have this growing box

File: shared/settings/invites/index.desktop.js [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+        style={{color: globalColors.red}}
+      >
+        Reclaim
+      </Text>
+    </Box>
+  )
+}
+
+function PendingEmailContent ({invite}: {invite: PendingEmailInvite}) {
+  console.log(invite)
```
Comment (chrisnojima) [inline shared/settings/invites/index.desktop.js]: 🔥

File: shared/settings/invites/index.desktop.js [hunk 3]
```diff
@@ -0,0 +1,150 @@
+// @flow
+import React from 'react'
```
Comment (chrisnojima) [inline shared/settings/invites/index.desktop.js]: lets sort these

File: shared/settings/invites/index.desktop.js (hunk 2 above)
Comment (chrisnojima) [inline shared/settings/invites/index.desktop.js]: this should be fixed before the merge but the other stuff is 🐑

Comment (chromakode) [inline shared/settings/invites/index.desktop.js]: Whoops! :smile:

**Comment by keybase-ci-visdiff** [comment]
:mag_right: The commits e4fea7c30ce7...f8d04afe4d23 introduced some visual changes on linux:
 * Added: **Invites-Normal_Empty_Message**  
   Invites-Normal_Empty_Message rendered
 * Added: **Invites-Normal_Empty**  
   Invites-Normal_Empty rendered
 * Added: **Invites-Normal_Email_Error**  
   Invites-Normal_Email_Error rendered
 * Added: **Invites-Normal**  
   Invites-Normal rendered
 * Changed: **Landing-Normal_Not_Verified_email** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1475099235907-e4fea7c30ce7-f8d04afe4d23-linux/Landing-Normal_Not_Verified_email-e4fea7c30ce7.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1475099235907-e4fea7c30ce7-f8d04afe4d23-linux/Landing-Normal_Not_Verified_email-f8d04afe4d23.png)  
   Landing-Normal_Not_Verified_email comparison
 * Changed: **Landing-Normal** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1475099235907-e4fea7c30ce7-f8d04afe4d23-linux/Landing-Normal-e4fea7c30ce7.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1475099235907-e4fea7c30ce7-f8d04afe4d23-linux/Landing-Normal-f8d04afe4d23.png)  
   Landing-Normal comparison
 * Changed: **Landing-Gold_Plan_Broken_Payment** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1475099235907-e4fea7c30ce7-f8d04afe4d23-linux/Landing-Gold_Plan_Broken_Payment-e4fea7c30ce7.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1475099235907-e4fea7c30ce7-f8d04afe4d23-linux/Landing-Gold_Plan_Broken_Payment-f8d04afe4d23.png)  
   Landing-Gold_Plan_Broken_Payment comparison
 * Changed: **Landing-Gold_Plan** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1475099235907-e4fea7c30ce7-f8d04afe4d23-linux/Landing-Gold_Plan-e4fea7c30ce7.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1475099235907-e4fea7c30ce7-f8d04afe4d23-linux/Landing-Gold_Plan-f8d04afe4d23.png) [(diff)](https://keybase-app-visdiff.s3.amazonaws.com/1475099235907-e4fea7c30ce7-f8d04afe4d23-linux/Landing-Gold_Plan.png)

**Pull request merged**